### PR TITLE
Add icon, desktop file, metainfo for easier packaging; use GtkApplication for better DE integration

### PR DESCRIPTION
**Pull request by soyersoyer**

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -21,10 +21,23 @@ void show_error(void)
...
+static void
+activate (GtkApplication *app,
+          gpointer        user_data)
+{
+	GtkWidget *window;
+
+	window = app_win_create(app);
+	gtk_widget_show_all(window);
+	gtk_window_present (GTK_WINDOW (window));
+}
```
Comment (tsmetana) [inline src/main.c]: Please don't mix the coding styles. You are apparently used to a different one, but I don't like the spaces between function name and parenthesis: `gtk_window_present (GTK_WINDOW (window));` and don't do the multiline function headers, unless the line gets too long. I know the GNOME apps look like this -- I find them harder to read. In any case -- the mix of styles is the worst option.

File: pkg/com.github.tsmetana.mpk3-settings.svg
Comment (tsmetana) [inline pkg/com.github.tsmetana.mpk3-settings.svg:1]: This looks nice. Not sure how well would it scale in menu, but larger one looks good.

File: src/main.c [hunk 2]
```diff
@@ -34,10 +47,11 @@ int main(int argc, char ** argv)
 		show_error();
 		return -1;
 	}
-	app_win = app_win_create();
-	gtk_widget_show_all(app_win);
-	gtk_main();
+	app = gtk_application_new ("com.github.tsmetana.mpk3-settings", G_APPLICATION_DEFAULT_FLAGS);
+	g_signal_connect (app, "activate", G_CALLBACK (activate), NULL);
+  	status = g_application_run (G_APPLICATION (app), argc, argv);
+  	g_object_unref (app);
```
Comment (tsmetana) [inline src/main.c]: Dtto (spaces...).

**Review by tsmetana — CHANGES_REQUESTED** [review]
Apart from the cosmetic differences in the codind style, it's really good. Thank you.

File: src/main.c (hunk 1 above)
Comment (soyersoyer) [reply src/main.c]: Sorry, I thought my editor was smarter :)

File: pkg/com.github.tsmetana.mpk3-settings.svg
Comment (soyersoyer) [reply pkg/com.github.tsmetana.mpk3-settings.svg:1]: It looks good here. You can try it with:
```
desktop-file-install --dir="$HOME/.local/share/applications" --set-key=Exec --set-value="$PWD/src/mpk3-settings" --set-key=Path --set-value="$PWD" --set-key=Icon --set-value="$PWD/pkg/com.github.tsmetana.mpk3-settings.svg" pkg/com.github.tsmetana.mpk3-settings.desktop
```

**Review by tsmetana — APPROVED** [review]
Looks good now. Thanks again.

**Pull request merged**